src/frontend/src/views/Upload/components/Metadata/components/ImportFile/components/Instructions/index.tsx

```diff
@@ -23,14 +24,21 @@ export default function Instructions({ headers, rows }: Props): JSX.Element {
           included in your import file.
         </ListItem>
         <ListItem ordered fontSize="xs">
-          We recommend that you copy your metadata into our{" "}
+          We recommend that you copy your metadata into our
```

Comment:
`{" "}` wasn't being respected so i took it out, `StyledDownloadTemplate` Margin on the left was creating a sort of faux space